composer.json

```diff
@@ -55,7 +69,8 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "6.1-dev"
+            "dev-pcov": "6.1.4"
         }
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
Remove this as it'll only hurt resolution as it pulls everything in dev. It's not necessary if you explicitly require dev-... versions above.